README.md

```diff
@@ -2,11 +2,56 @@
 
 ## Getting Started
 
-1. Clone this repository
-2. Install packages with `npm install`
-3. Run with `npx tsx index.ts`
+1. Install npm package: 
 
-## Documentation
+```
+npm install git+https://github.com/irods/irods_client_http_typescript
+```
 
--   To browse the documentation for this library, open "docs/index.html"
--   If updates are made to the code, run "npx typedoc" to update the documentation html
+2. Import `Wrapper` class:
+
+```
+import { Wrapper } from 'irods_client_http_typescript'
+```
+
+3. Create a url configuration object, containing info about the irods http api that's running on your local machine:
+
+```
+const urlComponents: URLComponentsType = {
+    host: '<host>',
+    port: '<port>',
+    version: '<version>',
+}
+```
+
+4. Create a wrapper object, passing in the url components, as well as the username and password you want to log in with:
+```
+const api = new Wrapper(urlComponents, '<username>', '<password>')
+```
+
+5. Authenticate the wrapper object within an asynchronous context
+
+```
+(async () => {
+    await api.authenticate()
+})
+```
+
+
+- **Note**: This code is assuming you're not already within an asynchronous context, such as an async function. 
+
+6. All operations of the wrapper can now be done within an asynchronous context as well, e.g.,
+
+```
```

Comment:
Append `js` to all code blocks which show use of javascript.

That's how you tell GitHub markdown which language is being used. That adds syntax highlighting to the code block.

Reply:
Done.